pandas/tseries/frequencies.py

```diff
@@ -38,18 +38,20 @@ class FreqGroup(object):
     FR_NS = 12000
 
 
-US_RESO = 0
-MS_RESO = 1
-S_RESO = 2
-T_RESO = 3
-H_RESO = 4
-D_RESO = 5
+NS_RESO = 0
```

Comment:
Just learned they need to be available at the module level for some of the c imports, but inside the class they can live in a list, so I did that.